src/components/EmbeddedAskForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component, PropTypes } from 'react';
+
+import './EmbeddedAskForm.scss';
+
+export default class EmojiBarChart extends Component {
+  static propTypes = {
+    formScript: PropTypes.string
+  }
+
+  componentDidMount() {
+    this.injectScript();
+  }
+
+  shouldComponentUpdate() {
+    // Never re-render, it would clobber the embedded form!
+    return false;
+  }
+
+  injectScript() {
+    const { formScript } = this.props;
+
+    // Inspired by Google Analytics' script injection code:
+    // Use a pre-existing script tag as the model for the one we will insert
+    const scriptTag = document.getElementsByTagName('script')[0];
```

Comment:
Should this select an element by classname or id? this feels a bit brittle if someone goes and changes the order of things in the index.html file.